scripts/leapfrog/generate_resources.py

```diff
@@ -0,0 +1,322 @@
+#! /usr/bin/env python
+from argparse import ArgumentParser
+from ipaddress import IPv4Network
+from itertools import chain, izip, repeat, islice
+from json import dumps
+import os
+from time import sleep
+
+from keystoneauth1.identity import v3
+from keystoneauth1 import session as session_
+from cinderclient import client as cinder_client
+from glanceclient import client as glance_client
+from neutronclient.v2_0 import client as neutron_client
+from novaclient import client as nova_client
+
+CLIENT_VERSION = "2"
```

Comment:
What I meant here is CLIENT_VERSION 2 is used for glance and nova, is there a risk this would be causing issues with different versions of a deploy (Kilo vs Liberty, ...)